controllers/bucketpool_controller.go

```diff
@@ -42,8 +42,7 @@ import (
 )
 
 const (
-	bucketPoolFinalizer        = "cephlet.onmetal.de/bucketpool"
-	bucketPoolSecretAnnotation = "ceph-client-secret-name"
+	bucketPoolFinalizer = "cephlet.onmetal.de/bucketpool"
```

Comment:
Shouldn't the group be `cephlet.api.onmetal.de` since `onmetal.de` is not under the `api` group?